build.gradle

```diff
@@ -9,12 +9,26 @@ plugins {
 apply plugin: 'java'
 
 ext {
-    assert JavaVersion.current() == JavaVersion.VERSION_1_8: "Set JAVA_HOME to JDK 8. Current version is ${JavaVersion.current()}"
+    isJava8 = JavaVersion.current() == JavaVersion.VERSION_1_8
     jsr308 = file(new File("..")).absolutePath
     cfPath = "${jsr308}/checker-framework"
     cfiPath = "${jsr308}/checker-framework-inference"
     afu = "${jsr308}/annotation-tools/annotation-file-utilities"
     picoPath = "${jsr308}/immutability"
+    compilerArgsForRunningPICO = [
```

Comment:
Instead of `PICO` I would probably use `CF`, to use the same name as in the CF build file. There is nothing PICO-specific.